### ISSUE TITLE
Crash in AttachmentGalleryActivity

### ISSUE THREAD

**Issue by spvince** [GetStream/stream-chat-android#2428]
**Describe the bug**
Crash in AttachmentGalleryActivity

**SDK version**
- 4.19.0

**To Reproduce**
Crash saw in firebase i'm not reproduce it

```
Caused by java.lang.UnsupportedOperationException: Binary XML file line #28: You must supply a layout_width attribute.
       at android.content.res.TypedArray.getLayoutDimension(TypedArray.java:779)
       at android.view.ViewGroup$LayoutParams.setBaseAttributes(ViewGroup.java:8147)
       at android.view.ViewGroup$MarginLayoutParams.<init>(ViewGroup.java:8339)
       at androidx.constraintlayout.widget.ConstraintLayout$LayoutParams.<init>(ConstraintLayout.java:2902)
       at androidx.constraintlayout.widget.ConstraintLayout.generateLayoutParams(ConstraintLayout.java:1934)
       at androidx.constraintlayout.widget.ConstraintLayout.generateLayoutParams(ConstraintLayout.java:486)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:865)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
       at io.getstream.chat.android.ui.databinding.StreamUiActivityAttachmentGalleryBinding.inflate(StreamUiActivityAttachmentGalleryBinding.java:87)
       at io.getstream.chat.android.ui.databinding.StreamUiActivityAttachmentGalleryBinding.inflate(StreamUiActivityAttachmentGalleryBinding.java:81)
       at io.getstream.chat.android.ui.gallery.AttachmentGalleryActivity.onCreate(AttachmentGalleryActivity.kt:77)
       at android.app.Activity.performCreate(Activity.java:7327)
       at android.app.Activity.performCreate(Activity.java:7318)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3096)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3259)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1950)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7073)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```

**Device:**
 - Vendor and model:  Samsung s7
 - Android version: 8

**Comment by zsmb13** [comment]
Hi @spvince! Thanks for reporting the issue, we'll look into it.

Are you doing any custom theming for Stream Views in your application?

**Comment by spvince** [comment]
Hi @zsmb13, no custom theming

**Comment by adasiewiczr** [comment]
hey @spvince,
we released a new version [4.20.0](https://github.com/GetStream/stream-chat-android/releases/tag/4.20.0) that contains the fix mentioned above.
